apps/server/server.cpp

```diff
@@ -33,12 +33,17 @@
 #include "aditof/aditof.h"
 #include "aditof/sensor_enumerator_factory.h"
 #include "aditof/sensor_enumerator_interface.h"
+#ifdef PROTOBUF_DEFINED
 #include "buffer.pb.h"
-
+#endif
 #include "../../sdk/src/connections/utils/connection_validator.h"
 
 #include <csignal>
+#ifdef GLOG_DEFINED
```

Comment:
We can keep glog. There is no reason to let go of glog if we only disabled it on PC.